deploy/workloads/gen-benchmark-rootfs.py

```diff
@@ -84,7 +84,7 @@ def generate_init_script(command):
     init_script_body = init_script_head + "    " + command + init_script_tail
 
     temp_script = BUILD_DIR + "/temp"
-    with open(temp_script, 'wb') as f:
+    with open(temp_script, 'w') as f:
```

Comment:
why was the 'b' removed?

Reply:
IIRC this created a weird init script that wouldn't run. In the end, this should be fine because we are just writing normal strings into the file (not a binary encoded string `b"string here"`).